spec/acceptance/suites/default/50_grafana_spec.rb

```diff
@@ -363,13 +364,6 @@ class { 'simp_grafana':
     it 'is idempotent' do
       apply_manifest_on grafana, manifest, :catch_changes => true
     end
-
```

Comment:
The original author put this test in for a reason.  Is it no longer needed?

Reply:
It's been a while, but I think the test was added here out of convenience-- due to time-constraints on the initial work.  It can probably be moved to [simp_elasticsearch](/simp/pupmod-simp-simp_elasticsearch).  On the other hand, I might have added it as a sanity check to fail early in case things weren't all fine and dandy.  The full test suite took around 45minutes, IIRC, so identifying failure very early was a high priority.

Reply:
@brandonrdn Do you not need it as a sanity check, an [assertion](http://wiki.c2.com/?WhatAreAssertions)?